core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/models.ts

```diff
@@ -107,3 +108,36 @@ export interface ToolbarProps {
         hideSocialMedia: boolean;
     };
 }
+
+/**
+ * This is used for model the props of
+ * the New UVE Toolbar with Preview Mode and Future Time Machine
+ *
+ * @export
+ * @interface UVEToolbarProps
+ */
+export interface UVEToolbarProps {
+    editor: {
+        bookmarksUrl: string;
+        copyUrl: string;
+        apiUrl: string;
+    };
+    preview?: {
+        deviceSelector: {
+            apiLink: string;
+            hideSocialMedia: boolean;
+        };
+    };
+    personaSelector: {
+        pageId: string;
+        value: DotPersona;
+    };
+    runningExperiment?: DotExperiment;
+    currentLanguage: DotLanguage;
+    workflowActionsInode?: string;
+    unlockButton?: {
+        inode: string;
+        loading: boolean;
+    };
+    showInfoDisplay?: boolean;
+}
```

Comment:
Are we sure we want one state object for the two toolbar? I am seeing like we have 2 toolbar now, aren't we?

Reply:
I think no, so each toolbar has its state with a different object (similars, but in a different structure)
This is the object of the new toolbar